components/youtube/new-videos-by-username.js

```diff
@@ -0,0 +1,84 @@
+const axios = require("axios");
+const youtube = require("https://github.com/PipedreamHQ/pipedream/components/youtube/youtube.app.js");
+
+module.exports = {
```

Comment:
Would you mind fixing the indentation on this file? Note that the other files use an indentation of 2 spaces, this one indents with 4 spaces.

We're planning to add global ESLint / Prettier rules to this repo so that your editor should inherit those. That way future updates should be easier.